openslides/core/views.py

```diff
@@ -24,6 +26,18 @@
 from .serializers import CustomSlideSerializer, TagSerializer
 
 
+class AngularView(utils_views.View):
+    def get(self, *args, **kwargs):
+        """
+        Returns the angular base template.
+
+        As default the template lies in openslides/core/static/templates/index.html
+        """
+        with open(finders.find('templates/index.html')) as f:
+            content = f.read()
+        return HttpResponse(content)
```

Comment:
I think we need a authentication check so that if the user is anonymous and guest users are disabled there should be a redirect to the login page.